pkg/controller/controller.go

```diff
@@ -630,6 +632,67 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 			cont.opflexDeviceDeleted(dn)
 		})
 	go cont.apicConn.Run(stopCh)
+	go cont.enableGlobalSync(stopCh)
```

Comment:
Is there a reason why this doesn't get called after here instead:
https://github.com/noironetworks/aci-containers/blob/master/pkg/controller/environment.go#L290